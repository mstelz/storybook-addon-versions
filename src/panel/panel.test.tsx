import React from 'react';
import renderer, { act } from 'react-test-renderer';

import Panel from './';

jest.mock('../utils/config');

const location = {
  pathname: '/0.2.5/',
  hash: '',
  search: '',
};

describe('Panel', () => {
  it('renders correctly, no versions', () => {
    const tree = renderer
      .create(<Panel location={location} active={true} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  xit('renders versions', () => {
    let tree;
    act(() => {
      tree = renderer
        .create(<Panel location={location} active={true} />)
        .toJSON();
    });
    expect(tree).toMatchSnapshot();
  });
});
