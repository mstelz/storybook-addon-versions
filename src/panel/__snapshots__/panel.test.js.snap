// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders correctly, no versions 1`] = `
<Panel
  location={
    Object {
      "hash": "",
      "pathname": "/0.2.5/",
      "search": "",
    }
  }
  storybook={
    Object {
      "getQueryParam": [Function],
      "setQueryParams": [Function],
    }
  }
/>
`;

exports[`Panel renders versions 1`] = `null`;
